src/command_lint.cc

```diff
@@ -47,20 +49,46 @@ auto sourcemeta::jsonschema::cli::lint(
                                  sourcemeta::core::schema_format_compare);
       output << "\n";
     }
+
+    // If --fix and --json were both requested, optionally print a small JSON:
```

Comment:
In similar cases, other linters will output exactly the same, just that if you run it again, there will be no issues. Let's adopt that, as developers will be familiar with that